### ISSUE TITLE
Mu does not support Chinse cha

### ISSUE THREAD

**Issue by yonghuming** [mu-editor/mu#376]
If you are reporting a bug, we would like to know:

i want to print chinese char,
![image](https://user-images.githubusercontent.com/435079/37144292-8afe9ffe-22f8-11e8-98d9-134cb65a2b2c.png)
```
# -*- coding:utf-8 -*-
print("hello world!你好")
```
but i got an error
```
  File "c:\users\langxianmeng\mu_code\zh_cn_demo.py", line 2
SyntaxError: (unicode error) 'utf-8' codec can't decode byte 0xc4 in position 12: invalid continuation byte
>>> 
```

**Comment by yonghuming** [comment]
it seems that mu saved file as ANSI format so do not suppot chinese even if i declare # -*- coding:utf-8 -*-

**Comment by tjguk** [comment]
Thanks, @yonghuming this is being addressed in PR #370, which is in progress. Please feel free to comment over there. (And I'm very glad to have a non-Western tester as well!)

**Comment by tjguk** [comment]
Duplicate of #370

**Comment by yonghuming** [comment]
@tjguk think you i think we need to change code dealing with read/write stuffs

**Comment by carlosperate** [comment]
Out of curiosity @yonghuming, could you create the same python file with notepad and attach it to this issue? (maybe the file will have to have a `.txt` extension https://help.github.com/articles/file-attachments-on-issues-and-pull-requests/).

**Comment by ntoll** [comment]
This was fixed by #399
